tasks.py

```diff
@@ -27,15 +27,15 @@
     iterable=["component", "docker_args"],
     help={
         "component": "The components to build - if none given defaults to: "
-                     + ", ".join(ALL_COMPONENTS),
+        + ", ".join(ALL_COMPONENTS),
         "dry_run": "Do not perform any changes, just generate configs and log what would be done",
         "docker_args": (
-                "Arguments to build docker imaages --docker-args foo=bar. "
-                + "Repeat for multiple build arguments."
+            "Arguments to build docker images --docker-args foo=bar. "
+            + "Repeat for multiple build arguments."
```

Comment:
Being plain strings no `+` is required to concatenate them `"" ""` is valid as well - what do you think, should we use that instead?

Reply:
Ideally, I'd want to see it formatted like this:
```
        "docker_args": "Arguments to build docker images --docker-args foo=bar. "
                       "Repeat for multiple build arguments.",
```
But black considers that it should rather be like this:
```
        "docker_args": "Arguments to build docker images --docker-args foo=bar. "
        "Repeat for multiple build arguments.",
```
Without the indent it's hard to separate keys from continuation of values in the dictionary, so I think the original `+` makes it slightly more readable. But the extra set of `()` around the string could be avoided. Updated according to that.